eZ/Publish/Core/Persistence/Legacy/Content/Type/Gateway/DoctrineDatabase.php

```diff
@@ -1540,4 +1541,89 @@ public function removeFieldDefinitionTranslation(
 
         $deleteQuery->execute();
     }
+
+    /**
+     * Removes types created or modified by the user.
+     */
+    public function removeByUserAndVersion(int $userId, int $version): void
+    {
+        $this->connection->beginTransaction();
+
+        $qb = $this->connection->createQueryBuilder();
```

Comment:
Why can't you call the variable `$queryBuilder`. We should avoid using abbreviations.